### DISCUSSION TITLE
Debug Warning due to plots made by multiple main modes using a same keys

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#2947] [Q&A]
I had been plotting using two main nodes on different network using a same wallet keys. When I started to migrate to one main nodes with more than one harvester (converting the other main node to a pure harvester), the log file indicated a warning,

2021-04-27T15:28:36.539 harvester chia.plotting.plot_tools: WARNING  Plot /Users/USER/DIRECTORY/plot-k32-2021-xxxxx.plot has a farmer public key that is not in the farmer's pk list.

Is this a problem?

**Comment by dorkmo** [comment]
is it just showing that for the single plot? kinda wierd. might just delete it and see what happens.

**Reply by pluto2021** [reply]
Got two plots on that computer before. After the migration to farmer mode, both plots show the same warning in the GUI. Even though I intend to run it as a farmer only (upnp disabled in config file as directed), I am still using the GUI to schedule the plots. When I check the plots in the command shell, they are both valid. Will try to copy to the main node manually and see if they get recognized.

**Reply by dorkmo** [reply]
im not experienced in moving plots and runnning harvesters, but maybe try putting your 24 words into the harvester again?
